design_activity.md

```diff
@@ -10,19 +10,20 @@ While we encourage you to use the provided design for this project, there is muc
 - Compare and contrast different designs for the same program
 - Implement a program from a provided UML diagram
 
-## Your Design
+## First Draft Design
 
-Read through the project requirements with your partner, then follow our standard [object-oriented design process](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/02-intermediate-ruby/oo-design.md).
+1. Read through the project requirements with your partner, then follow our standard [object-oriented design process](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/02-intermediate-ruby/oo-design.md).
+    - What are the nouns, and how are they composed into objects and relations?
```

Comment:
Do we want to include a question about what properties the nouns have and/or split the relation and object parts into two sub bullets?